src/components/NcPopover/NcPopover.vue

```diff
@@ -370,7 +567,7 @@ export default {
 		},
 
 		async afterShow() {
-			this.getPopoverContentElement().addEventListener('transitionend', () => {
+			this.getPopoverContentElement()?.addEventListener('transitionend', () => {
```

Comment:
Why?

Reply:
Can be undefined. Is reported in the console directly after mount

Reply:
Are you sure this is expected behavior, that it is undefined here?

If it is - we probably need to do everything in this function when it's actually ready.
If it isn't - we needto prevent it from happening (and use `!.`)

Reply:
its not typescript so no `!` operator. But I checked its only the other place that is affected. so reverted this change.